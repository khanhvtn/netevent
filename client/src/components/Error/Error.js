import { Box, Grid, Hidden, Typography } from '@material-ui/core';
import React from 'react';
import makeStyles from './styles';

const Error = () => {
    const css = makeStyles();
    return (
        <>
            <Box className={css.errorBox} >
                <Typography
                    align="center"
                    variant="caption"
                >
                    404: Page Not Found...
                </Typography>
            </Box>
        </>
    );
};

export default Error;