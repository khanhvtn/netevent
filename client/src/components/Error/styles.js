import { makeStyles } from '@material-ui/core/styles';

export default makeStyles((theme) => ({
    errorBox: {
        backgroundColor: 'background.default',
        display: 'flex',
        flexDirection: 'column',
        height: '100%',
        justifyContent: 'center',
        marginTop:200
    },
    imageAlign: {
        textAlign: 'center'
    },
    handleImage: {
        marginTop: 80,
        display: 'inline-block',
        maxWidth: '100%',
        width: 560,
        [theme.breakpoints.down('sm')]:{
            marginTop: 10
        }
    },
    pnf:{
        fontWeight: 'bold',
        [theme.breakpoints.down('sm')]:{
            fontSize: '4em'
        }
    }
}));

