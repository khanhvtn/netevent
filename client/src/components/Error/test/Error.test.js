import React from 'react'
import { create } from 'react-test-renderer'
import expect from 'expect'

import Error from '../Error'

describe('Error Component', () => {
    test('Matches the snapshot', () => {
        const error = create(<Error />)
        expect(error.toJSON()).toMatchSnapshot()
    });
});