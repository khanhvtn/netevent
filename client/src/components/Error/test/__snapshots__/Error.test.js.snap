// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component Matches the snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-4 makeStyles-errorBox-1"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthMd"
  >
    <h1
      className="MuiTypography-root MuiTypography-h1 MuiTypography-colorTextPrimary MuiTypography-alignCenter"
    >
      404: Page Not Found
    </h1>
  </div>
</div>
`;
